src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.RoslynWorkspace/RazorWorkspaceListener.cs

```diff
@@ -45,17 +50,38 @@ public void Dispose()
         _disposeTokenSource.Dispose();
     }
 
-    public void EnsureInitialized(Workspace workspace, string projectInfoFileName)
+    public void EnsureInitialized(Workspace workspace, string pipeName)
+    {
+        // Make sure we don't hook up the event handler multiple times
```

Comment:
Nit: Comment is wrong, there is no event any more. We'll ignore the fact there wasn't before this change either :)

Reply:
I thought it was about `WorkspaceChanged` events?

Reply:
Oh.. yeah that makes sense. I didn't notice that this calls the other initialize method 🤦‍♂️